README.md

```diff
@@ -2,29 +2,29 @@
 
 # Databraid Sheet API
 
-A part of the *Databraid Dashboard*
 ## Description
-This is an API for the Google Sheets widget. It represents a microservice that functions indepently or as part of the [Databraid Dashboard](https://github.com/databraid-dashboard/dashboard-spa). The Sheets widget leverages [GraphQL](http://graphql.org/) to create a powerful and fast singular endpoint to access the [Google Sheets API](https://developers.google.com/sheets/api/). With this widget, you can project your most relevant Google Spreadsheet file into your dashboard and keep up with changes in real time.  
+This is an API for the Google Sheets widget. It represents a microservice that functions in conjunction with the [Sheets Single Page Application](https://github.com/databraid-dashboard/sheet-spa) or as part of the [Databraid Dashboard](https://github.com/databraid-dashboard/dashboard-spa). This API leverages [GraphQL](http://graphql.org/) to create a powerful and fast singular endpoint to access the [Google Sheets API](https://developers.google.com/sheets/api/). With this widget, you can project your most relevant Google Spreadsheet file into your dashboard and keep up with changes in real time.  
 ## Usage
-If you have a google account, you're well on your way! Please fork and clone this repository, before beginning make sure you have Docker installed and running, then from the command line:
+
+If you have a Google account, you're well on your way! Please fork and clone this repository, before beginning make sure you have Docker installed and running, then from the command line:
 
 ```
 npm i
 npm run up
 ```
-This will compose your API from the latest docker image into a docker container available to you locally via port 8003 and will map to an exposed container port of 8000. Please note that when developing, your local requests should be sent through port 8003. This can be modified in the docker-compose.yml should you have need to change the host to container mapping.
+This will compose your API from the latest docker image into a docker container available to you locally via port ```8003``` and will map to an exposed container port of ```8000```. Please note that when developing, your local requests should be sent through port ```8003```. This can be modified in the ```docker-compose.yml``` should you have need to change the host to container mapping.
```

Comment:
@Gwillison415 for inline fixed width you only need to use one backtick. The 3 here are when you need to do multiple lines.

Reply:
interesting, it seems to work in my markdown preview but  i did not know that part and will change it!